PL2/PL2-gui/src/main/java/nl/tudelft/pl2016gr2/gui/view/graph/ViewAnnotation.java

```diff
@@ -62,9 +71,35 @@ public void addLabel(ArrayList<Node> nodes) {
   }
 
   /**
+   * Add an arrow to indicate the annotation direction.
+   *
+   * @param nodes the list of nodes to which to add the arrow.
+   */
+  public void addArrow(ArrayList<Node> nodes) {
+    if (getWidth() < MINIMUM_ARROW_WIDTH || getHeight() < MINIMUM_LABEL_HEIGHT) {
+      return;
+    }
+    ImageView arrow;
+    if (Objects.equals(annotation.strand, "-")) {
```

Comment:
Did not explicitly realise, but my point was that deciding to draw either forward or backward arrows should be done with different annotation(view) classes. the null-check is good for safety right now but in general shouldn't be necessary